main/modes/mode_flight.c

```diff
@@ -1562,18 +1531,96 @@ static void flightTimeHighScoreInsert( int insertplace, bool is100percent, char
 //////////////////////////////////////////////////////////////////////////////
 // Multiplayer
 
+static void FlightNetworkDraw(uint32_t now, struct ModelRangePair ** mrp );
+{
+	struct ModelRangePair * mrptr = *mrp;
+
+	flight_t * flt = flight;
+
+	if( flt->nNetworkServerLastSeen > 0 ) flt->nNetworkServerLastSeen--;
+
+	{
+		multiplayerpeer_t * p = flt->allPeers
+		multiplayerpeer_t * end = p + MAX_PEERS;
+		for( ; p != end; p++ )
+		{
+			int ticks = p->validForTicks;
+			if( ticks )
+			{
+				// Render peer/ship.
+
+				int32_t deltaTime = now - p->timeOfUpdate;
+	// Not valid for server.
+	uint32_t timeOfUpdate;  // In our timestamp
```

Comment:
You may want to run this through an autoformatter to at least normalize tabs, spaces, and the like